tests/unittests/tools/stub_http_download.c

```diff
@@ -17,6 +17,7 @@ typedef struct http_download_t
     ProfWin* window;
     pthread_t worker;
     int cancel;
+    int silent; // gboolean
```

Comment:
We can use int here or we can include the glib header that provides gboolean. Or why did you use int?